src/azure-cli/HISTORY.rst

```diff
@@ -35,9 +35,11 @@ Release History
 * vm create: Add --enable-agent configuration.
 * vm create: Use standard public IP SKU automatically when using zones.
 * vm create: Compose a valid computer name from VM name if computer name is not provided.
+* vm create: Add --workspace to enable log analytics workspace automatically.
+* vm extension set: Move --name and --extension-instance-name outside Resource Id Arguments group.
```

Comment:
Maybe clarify with something like: "Fix bug where users could not set an extension on a vm with --ids" or "Fix bug where --ids expects an extension subresource instead of a vm resource."

Or whichever combination makes sense!

Reply:
Good advice! Thanks!